src/tools/pub/publishing.md

```diff
@@ -39,10 +39,11 @@ with your package. In both cases, pub tries to help you by pointing out what
 changes will help make your package play nicer with the Dart ecosystem. There
 are a few additional requirements for uploading a package:
 
-* You must include a `LICENSE` file
-  that contains an [open-source license.][open-source license]
-  We recommend the [BSD license,][BSD license]
-  which is used by Dart itself. You must also have the legal right to
+* You must include a `LICENSE` file.
+  We recommend the [BSD 3-clause license][],
+  which the Dart and Flutter teams use.
```

Comment:
s/use/typically uses/.

(We have a few projects that use other licenses, like the MIT license, for historical reasons)